cdap-docs/examples-manual/source/examples/wikipedia-data-pipeline.rst

```diff
@@ -88,6 +96,28 @@ This application demonstrates:
 .. include:: _includes/_building-starting-running.txt
 
 
+Deploying the Example
+=====================
+Since deploying the WikipediaPipelineApp involves loading an artifact and creating an application from it, the
+preferred method of deploying it is to use the CDAP CLI.
+
+- Load the Artifact::
+
+    $ cdap-cli.sh load artifact target/WikipediaPipeline-3.3.0-SNAPSHOT.jar name WikipediaPipelineApp version

```

Comment:
It does work. The ``config-file`` param seems optional. In this case, I do not need a config that specifies parents etc.

```
cdap (http://localhost:10000/default)> load artifact ~/work/git/cdap/cdap-examples/WikipediaPipeline/target/WikipediaPipeline-3.3.0-SNAPSHOT.jar name WikipediaPipelineApp version 3.3.0-SNAPSHOT
Successfully added artifact with name 'WikipediaPipelineApp'

cdap (http://localhost:10000/default)> load artifact ~/work/git/cdap/cdap-examples/WikipediaPipeline/target/WikipediaPipeline-3.3.0-SNAPSHOT.jar
Successfully added artifact with name 'WikipediaPipeline'

cdap (http://localhost:10000/default)> load artifact ~/work/git/cdap/cdap-examples/WikipediaPipeline/target/WikipediaPipeline-3.3.0-SNAPSHOT.jar name WikipediaA version 3.4.0-SNAPSHOT
Successfully added artifact with name 'WikipediaA'

cdap (http://localhost:10000/default)> list artifacts user
+===============================================+
| name                 | version        | scope |
+===============================================+
| WikipediaA           | 3.4.0-SNAPSHOT | USER  |
| WikipediaPipeline    | 3.3.0-SNAPSHOT | USER  |
| WikipediaPipelineApp | 3.3.0-SNAPSHOT | USER  |
+===============================================+
```